DataSource/Util/ChoicesBuilder.php

```diff
@@ -32,23 +32,31 @@ public function build($choicesConfiguration)
 
         if (
             is_array($choicesConfiguration) &&
-            isset($choicesConfiguration['repository']) &&
-            isset($choicesConfiguration['field'])
+            isset($choicesConfiguration['repository'])
         ) {
             if ($choicesConfiguration['repository'] instanceof EntityRepository) {
```

Comment:
maybe put this if section inside a method like ` $this->getChoicesFromRepository` to make it a bit more readable? you can then return early after each if.